srcpkgs/OpenJK/INSTALL.msg

```diff
@@ -0,0 +1 @@
+Consult /usr/share/doc/OpenJK/README.voidlinux for further installation instructions.
```

Comment:
remove this

Reply:
Will do, is the readme okay to keep in there?

Reply:
the readme is fine to keep, but nobody reads `INSTALL` messages